libs/blocks/quiz-marquee/quiz-marquee.css

```diff
@@ -0,0 +1,184 @@
+.quiz-marquee {
+  padding: var(--spacing-m) 0;
+  position: relative;
+}
+
+.dark .quiz-marquee,
+.quiz-marquee.dark {
+  color: white;
+}
+
+.quiz-marquee .static p { margin: var(--spacing-xs) 0; }
+.quiz-marquee .static p:only-child { margin: 0; }
+.quiz-marquee .static p:first-child { margin-top: 0; }
+.quiz-marquee .static [class^="heading"] { margin: 0 0 var(--spacing-xs) 0; }
+.quiz-marquee .static [class^="detail"] { margin: 0 0 var(--spacing-xxs) 0; }
+.quiz-marquee .static *:last-child { margin-bottom: 0; }
+
+.quiz-marquee > .foreground {
+  max-width: var(--grid-container-width);
+  margin: 0 auto;
+  display: grid;
+  gap: var(--spacing-m);
+}
+
+.quiz-marquee > .foreground.fw {
+  width: 100%;
+}
+
+.quiz-marquee .foreground .copy {
+  display: grid;
+  gap: var(--spacing-xs);
+}
+
+.quiz-marquee hr {
+  width: 100%;
+  border: none;
+  height: 1px;
+  margin: 0;
+}
+
+.quiz-marquee .has-divider {
+  margin: var(--spacing-xxs) 0;
+}
+
+.quiz-marquee .badge {
+  font-size: 14px;
+  line-height: 1em;
+  font-weight: 400;
+  vertical-align: middle;
+  padding: 4px 8px;
+  margin-left: 4px;
+  border-radius: 4px;
+  background-color: yellow;
```

Comment:
Is there a var or specific hex we should be using?

Reply:
Good call. I'll check the Design, Yellow is pretty vague.

Reply:
@fullcolorcoder  - If you look at your url, notice the /delta/ path. That is wrong. 

Clear your cache and try the quiz again and it should reference the correct path. `/drafts/rparrish/quiz/rec/` etc
https://rparrish-quiz-marquee--milo--adobecom.hlx.page/drafts/rparrish/quiz/rec/app-recommender-test/

I am noticing something changed w/ the quiz block and that block isn't resolving now. Hold please, I'll updated that page and let you know. 

@Axelcureno Do you know if there has been changes to the document of the quiz block?

Reply:
Ok, i compared some examples and it looks like the `quiz` block document model has changed. I've updated my draft and it's working now. Ping me if you have any questions. Thanks